src/index.js

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import MapSection from "./map";
+import './style.css';
 
 ReactDOM.render(
-  <h1>Hello World!</h1>,
+  <MapSection> </MapSection>,
```

Comment:
You can save yourself a couple characters by just saying `ReactDOM.render(<MapSection />, . . . )` All React elements can have self closing tags.